packages/styles/src/sass/components/banner/_usa.scss

```diff
@@ -1,148 +1,101 @@
-//@use '../../base/colors' as *;
 @use '../../grid/breakpoints' as *;
 
-.cbp-banner {
-  background-color: var(--cbp-color-base-neutral-darkest);
-  width: 100%;
+:root {
+  --cbp-usa-banner-color: var(--cbp-color-text-lightest);
+  --cbp-usa-banner-background-color: var(--cbp-color-base-neutral-darkest);
+}
 
-  &__title {
-    align-items: center;
-    display: flex;
-    height: 52px;
+[data-variant=light] {
```

Comment:
Do we need this variant?

Reply:
deleted. I was just trying out how easy it would be to make the light variants with a sort of higher-level API. I'm still going to leave the higher-level CSS vars.